### DISCUSSION TITLE
part-ii/2019-101/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#305] [Question comments]
Part II 2019 Number Theory: Paper 2, Section I, I 

https://questions.tripos.org/part-ii/2019-101/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

Pretty much all bookwork, with a standard method at the end using various results about Jacobi/Legendre symbols. I got $(\tfrac{503}{2019})=-1$.

</details>
